src/applications/facility-locator/config.js

```diff
@@ -2,6 +2,11 @@
 import environment from '../../platform/utilities/environment';
 import isProduction from '../../platform/utilities/environment/isProduction';
 
+// TODO: Remove me when done bug fixing
```

Comment:
can this be removed now?

Reply:
Not as of yet, no. I had originally intended to check in a few more fixes to this branch before opening the PR, but in the interest of speeding things up on our end I opened this up as-is. I'll be opening another PR for later fixes but left that line in to remind me for when I don't need to connect to somewhere else for API reasons (e.g. running UI locally but connecting to a RI API).